docs/Contribute/ContributeDesign.md

```diff
@@ -0,0 +1,10 @@
+# Design BTCPay Server
+
+## Why designing matters
+
+Designing for BTCPay Server consists of creating and refining the user interface (UI) and user experience (UX) on a viewing standpoint, making it easier to navigate the menus, options, etc...
```

Comment:
> making it easier to navigate the menus, options, etc...

Not a fan of this. `improving  overall experience for the end-user` sounds better imo

Reply:
```suggestion
Designing for BTCPay Server consists improving overall experience for the end-user, making it easier to navigate the menus, options, etc...
```